charts/hush-sensor/values.yaml

```diff
@@ -122,6 +122,12 @@ daemonSet:
   # https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
   tolerations: []
 
+  # An additional set of volumes
+  extraVolumes: []
+
+  # An additional set of mounts for sensor
+  sensorExtraVolumeMounts: []
```

Comment:
why admin has to specify both extraVolumes and sensorExtraVolumeMounts?

also, why one variable starts with sensor and the other doesnt?

Reply:
The name prefix is different because `extraVolumes` affects the whole `DaemonSet` while `sensorExtraVolumeMounts` affects `sensor` only.

The values must be separate because the schema for `Volumes` and `VolumeMounts` in k8s is different.

Reply:
ack

Reply:
IOW if in future we will need to support mounting an extra volume into `sensor-vector` the volume definition goes to `extraVolumes` together with volumes for `sensor`. While volume mounts will go to `sensorVectorExtraVolumeMounts`.